<launch>

    <!-- Commenting this out until we know that dilation is working correctly!! -->
    <node pkg="mapping" type="mapping" name="mapping" respawn="true">
    	<param name="length_in_x" value ="70.0"/>
    	<param name="length_in_y" value="70.0"/>
    	<param name="resolution" value ="0.5"/>
    	<param name="min_variance" value="0.0009"/>
    	<param name="max_variance" value="0.09"/>
    	<param name="mahalanobis_distance_threshold" value="0.5"/> <!-- 0.8 -->
    	<param name="min_horizontal_variance" value=".125"/>
    	<param name="max_horizontal_variance" value="0.5"/>
    	<param name="multi_height_noise" value="0.0009"/>
    	<param name="min_update_rate" value = "5.0"/>
    	<param name="fused_map_publishing_rate" value="5.0"/>
    	<param name="sensor_processor/type" value="Laser"/>
    	<param name="surface_normal_estimation_radius" value="0.990"/>
    	<param name="/sensor_processor/ignore_points_above" value="2.750"/>
    	<param name="/sensor_processor/ignore_points_below" value="-1.0"/>
        <param name="/sensor_processor/min_radius" value="0.0001" />
        <param name="/sensor_processor/beam_angle" value="0.001" />
        <param name="/sensor_processor/beam_constant" value="0.01" />
    	<param name ="point_cloud_topic" value="/back_lidar_points"/>
    </node>

    <node pkg ="mapping" type="map_build_node" name="map_build_node" respawn="true" output="screen">
    	<param name="base_height" value="0.40"/>
    	<param name="robot_width" value="1.4"/>
    	<param name="erode_size" value= "2"/>
    	<param name="map_topic" value="/mapping/ele_map"/>
    </node>

</launch>
